yeezyCopper.js

```diff
@@ -60,69 +64,153 @@ function pay () {
     }
   });
 
-  $('input#dwfrm_payment_creditCard_cvn').val(function () {
+  $('input#dwfrm_payment_creditCard_cvn').val(function() {
     return config.security_code;
   });
 
-  setTimeout(function () {
+  setTimeout(function() {
     placeOrder();
   }, 300);
 }
+
+function _x(STR_XPATH) {
```

Comment:
_x is a bit of an obscure name, no idea what this function does. maybe getXNodes or something. also parameter names are typically camelCase and ALL_CAPS_VARIABLES denote globals (generally speaking although not a rule).